gradle/libs.versions.toml

```diff
@@ -71,6 +71,7 @@ jakarta-ws-rs-api = { module = "jakarta.ws.rs:jakarta.ws.rs-api", version = "4.0
 javax-servlet-api = { module = "javax.servlet:javax.servlet-api", version = "4.0.1" }
 junit-bom = { module = "org.junit:junit-bom", version = "5.12.2" }
 logback-classic = { module = "ch.qos.logback:logback-classic", version = "1.5.18" }
+kryo = { module = "com.esotericsoftware:kryo", version = "5.5.0" }
```

Comment:
Just flagging that this is going to be controversial, somebody will have a serde that they swear is better and make a compelling case about why you should use it so be ready for that :)